connector/src/main/java/org/jboss/as/connector/subsystems/datasources/DataSourcesExtension.java

```diff
@@ -146,7 +148,7 @@ public class DataSourcesExtension implements Extension {
     public static final String SUBSYSTEM_NAME = Constants.DATASOURCES;
     private static final String RESOURCE_NAME = DataSourcesExtension.class.getPackage().getName() + ".LocalDescriptions";
 
-    static final ModelVersion CURRENT_MODEL_VERSION = ModelVersion.create(6, 0, 0);
+    static final ModelVersion CURRENT_MODEL_VERSION = ModelVersion.create(7, 2, 0);
```

Comment:
Note: this being at 6, 0, 0 instead of 7,0,0 was a serious bug.

Reply:
Well, a potentially serious bug. It seems 7.0.0 and 6.0.0 were equivalent in terms of domain mode, so the bug did no harm.